paper.md

```diff
@@ -123,7 +123,7 @@ julia> x = grid(D); u = sinpi.(x); # evaluate the function `sinpi` on the discre
 julia> du = D * u; # evaluate the discrete derivative of `u` using the SBP operator `D`
 
 julia> integrate(u -> u^2,
-         du - π * cospi.(x), D) |> sqrt # compute the discrete L² error of the approximation
+         du - pi * cospi.(x), D) |> sqrt # compute the discrete L² error of the approximation
```

Comment:
What was the matter with `π`? Did that not work when rendering the PDF? If possible, I prefer UTF8 for readability.

Reply:
The JOSS GitHub action threw an error: https://github.com/ranocha/SummationByPartsOperators.jl/runs/3105953836?check_suite_focus=true

Reply:
Ah ok, that can be resolved by either using the `jlcode` package or by introducing them as UTF8-symbols in code. But that is not your concern, maybe we (I) should open a JOSS issue for that, since I introduced quite a set of UTF8-stuff to jlcode some time back :)

Reply:
Yeah, opening a JOSS issues sounds great to me. Would you be willing to do so (since you can provide the relevant information, it seems)?

Reply:
I will do that the next days